concepticondata/conceptlists/Xiao-2012-213.tsv

```diff
@@ -0,0 +1,214 @@
+ID	NUMBER	CHINESE	PINYIN	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID	PEN_STROKES	ICONICITY	PICTURABILITY	CURRICULUM_ORDER	FREQ_PER_MILLION
+Xiao-2012-213-1	1	一	yi1	one	ONE	1493	1	0.65	5.55	3	12511.7
+Xiao-2012-213-2	2	八	ba1	eight	EIGHT	1705	2	0.2	5.84	9	588.4
+Xiao-2012-213-3	3	刀	dao1	knife	KNIFE	1352	2	0.2	6.74	32	531
+Xiao-2012-213-4	4	子	zi3	son	SON	1620	2	0.2	4.05	9	614.3
+Xiao-2012-213-5	5	卜	bu3	fortune telling			2	0.35	4.16	106.5	25.7
+Xiao-2012-213-6	6	了	liao3	to finish	FINISH	1766	2	0.4	2.39	15.5	8187.2
+Xiao-2012-213-7	7	力	li4	power	POWER	2526	2	0.4	2.39	89.5	2281.3
+Xiao-2012-213-8	8	厂	chang3	factory			2	0.45	5.39	106.5	631
+Xiao-2012-213-9	9	乃	nai3	therefore			2	0.45	1.39	212.5	25.2
+Xiao-2012-213-10	10	入	ru4	to enter	ENTER	749	2	0.5	2.74	61	1164.6
+Xiao-2012-213-11	11	又	you4	in addition			2	0.5	2.06	37.5	1192.9
+Xiao-2012-213-12	12	几	ji3	several			2	0.55	2.42	93	840.4
+Xiao-2012-213-13	13	七	qi1	seven	SEVEN	1704	2	0.6	5.77	9	892.7
+Xiao-2012-213-14	14	二	er4	two	TWO	1498	2	0.7	5.87	3	2049.8
+Xiao-2012-213-15	15	九	jiu3	nine	NINE	1483	2	0.75	5.68	9	935.5
+Xiao-2012-213-16	16	人	xiao3	person	PERSON	683	2	0.8	5.29	48.5	5835.6
+Xiao-2012-213-17	17	十	shi2	ten	TEN	1515	2	0.8	5.87	9	2021.8
+Xiao-2012-213-18	18	丁	ding1	man	MALE PERSON	2106	2	0.85	6.1	140.5	103.1
+Xiao-2012-213-19	19	丈	zhang4	to measure	MEASURE	1506	3	0.15	3	190	15.1
+Xiao-2012-213-20	20	弓	gong1	bow (as in bow and arrow)	BOW	994	3	0.25	6.23	32	14.8
+Xiao-2012-213-21	21	尸	shi1	dead body			3	0.25	5	206	12.8
+Xiao-2012-213-22	22	也	ye3	likewise			3	0.25	1.42	99	2348.3
+Xiao-2012-213-23	23	叉	cha1	fork	FORK	1302	3	0.3	6.81	195	54.4
+Xiao-2012-213-24	24	工	gong1	work	WORK (LABOUR)	984	3	0.3	3.03	72	4550.7
+Xiao-2012-213-25	25	千	qian1	thousand	THOUSAND	1843	3	0.3	4	113	545.4
+Xiao-2012-213-26	26	于	yu2	in, on, at			3	0.3	2.39	150	3546.4
+Xiao-2012-213-27	27	之	zhi1	it, him, her, them	HE OR SHE OR IT	262	3	0.35	1.97	173.5	1756.3
+Xiao-2012-213-28	28	丸	wan2	a small round ball			3	0.4	4.52	212.5	9.7
+Xiao-2012-213-29	29	大	da4	big	BIG	1202	3	0.4	4.2	15.5	6857.3
+Xiao-2012-213-30	30	己	ji3	oneself	SELF	1993	3	0.45	1.97	133	997.2
+Xiao-2012-213-31	31	寸	cun4	inch			3	0.5	4.52	75.5	268.2
+Xiao-2012-213-32	32	卫	wei4	to protect	PROTECT	2900	3	0.5	2.61	72	306.2
+Xiao-2012-213-33	33	习	xi2	be used to			3	0.5	1.35	83.5	1127.4
+Xiao-2012-213-34	34	才	cai2	ability	ABILITY	2527	3	0.5	1.58	140.5	613
+Xiao-2012-213-35	35	儿	er2	son	SON	1620	3	0.55	4.05	75.5	2499.1
+Xiao-2012-213-36	36	亡	wang2	dead	DEAD	1625	3	0.55	4.71	204	114.6
+Xiao-2012-213-37	37	么	mo	a word indicating a question			3	0.55	2.3	93	893.7
+Xiao-2012-213-38	38	万	wan4	million			3	0.55	3.23	122	648.6
```

Comment:
don't we have one million in concepticon? If not, I suggest to add it, also for the sake of the numerals projects. @chrzyki, can you check, is "one million" important? and can you advise how to add this to the meta-data?

Reply:
Yes, we have ONE MILLION in concepticon. I'll add the mapping.